<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterTechnicien</class>
 <widget class="QDialog" name="ajouterTechnicien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Technicien - Ajouter Technicien</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../build-connectbdd-Desktop_Qt_5_9_6_MinGW_32bit-Debug/logo-domo.png</normaloff>../../build-connectbdd-Desktop_Qt_5_9_6_MinGW_32bit-Debug/logo-domo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomTech">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string extracomment="test">Nom technicien</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTech">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Login technicien</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passTech">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Password technicien</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Générer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ajouterTechnicien</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ajouterTechnicien</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
